src/plugins/presentation_panel/public/panel_component/_presentation_panel.scss

```diff
@@ -106,12 +105,12 @@
   */
 
 .embPanel__optionsMenuButton {
-  background-color: transparentize($euiColorDarkestShade, .9);
+  background-color: transparentize($euiColorLightShade, .5);
   border-bottom-right-radius: 0;
   border-top-left-radius: 0;
 
   &:focus {
-    background-color: $euiFocusBackgroundColor;
+    background-color: transparentize($euiColorLightestShade, .5);
```

Comment:
There are four scenarios to consider for the colours of the `optionsMenuButton`:

1. Kibana is in light mode + the background of the image is light

| Before | After |
|--------|--------|
| Cell | Cell | 

2. Kibana is in light mode + the background of the image is dark
3. Kibana is in dark mode + the background of the image is light
4. Kibana is in dark mode + the background of the image is dark

This fixes scenarios 1
This fixes the above scenarios while keeping appearances more-or-less the **same** in following scenarios